[DOC] update to BEP process (#615)
bids-standard/bids-website: .github/PULL_REQUEST_TEMPLATE/bep.md

This is a suggested pull request template for to add BEP to the BIDS website.
It's designed to capture information we've found to be useful in reviewing pull requests.

If there is other information that would be helpful to include, please don't hesitate to add it!

---

> - [ ] indicate the issue in the BIDS specicification repository that you opened for your BEP.

Reviewer: ```suggestion
- [ ] indicate the issue in the BIDS specification repository that you opened for your BEP.
```

submitted:
- https://github.com/codespell-project/codespell/pull/3639

  BIDS specification issue: https://github.com/bids-standard/bids-specification/issues/????
- [ ] update the `data/beps/beps.yml` document using the template entry described [here](https://github.com/bids-standard/bids-website/tree/main/.github/PULL_REQUEST_TEMPLATE/bep_template.yml)

> - [ ] add information about BEP leads to the list of BIDS contributors in [the wiki of the BIDS specification](https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors#adding-yourself-as-a-contributor)

Reviewer: ```suggestion
- [ ] ensure that BEP leads are listed among BIDS contributors in [the wiki of the BIDS specification](https://github.com/bids-standard/bids-specification/wiki/Recent-Contributors#adding-yourself-as-a-contributor)
```

since they might already be there and thus 'add information' might no longer be pertinent?
